memphis.dev/README.md

```diff
@@ -0,0 +1,21 @@
+# Memphis.dev & Open Source
```

Comment:
@yanivbh1 seems like it's a new file. Need to make sure there's only one README.md in the folder.
Also, the banner should be used like this-
`<img src="https://raw.githubusercontent.com/Pull-Request-Community/open-source-rel/main/Memphis.dev/images/Memphis_PR_banner.png" alt="PR <3 Memphis.dev" width="100%"/>
`
Once we push the banner to the repo, we'll approve and merge the PR (after making sure these 2 changes introduced) and then it'll be completed :)

Reply:
@hilafish it's strange, I fetched all changes before the PR and still see only one file.
Regarding the banner - 404. File not found.

Reply:
Seems like I can delete the file from here, so add the change of the banner to the README.md, and then i'll delete the redundant file

Reply:
@hilafish I understood the issue. Should be ok now. Just need to add the banner.